.lintstagedrc

```diff
@@ -9,6 +9,6 @@
   "*.js": [
     "yarn run lint:css:fix",
     "yarn run lint:js:fix",
-    "yarn run test",
+    "yarn run test-lintstaged",
```

Comment:
I think there should be a space between test and -lintstaged

Reply:
this is full command name. Instead of run test system will run "test-lintstaged"

Reply:
Changed.